api/lib/infrastructure/repositories/membership-repository.js

```diff
@@ -0,0 +1,40 @@
+const BookshelfMembership = require('../data/membership');
+const { MembershipCreationError } = require('../../domain/errors');
+const { InfrastructureError } = require('../../infrastructure/errors');
+const Membership = require('../../domain/models/Membership');
+const Organization = require('../../domain/models/Organization');
+const OrganizationRole = require('../../domain/models/OrganizationRole');
+const bookshelfUtils = require('../utils/bookshelf-utils');
+
+function _toDomain(membershipBookshelf) {
```

Comment:
bookshelf-to-domain-converter ? :D

Reply:
Ca ne marche pas ici car les objets Bookshelf qui sont tirés via `withRelated` ou `load` n'ont pas d'attribut `type` et donc le _bookshelf-to-domain-converter_ ne peut pas faire de `new` avec le bon type. :-(